vendor/autoload.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
the vendor folder should not be in git, please remove all vendor related stuff - put in the gitignore